dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -116,18 +112,17 @@ public FileResourceStream getFileResourceImage(
       @Nonnull UID attribute,
       @CheckForNull UID program,
       ImageFileDimension dimension)
-      throws NotFoundException {
+      throws NotFoundException, ForbiddenException {
     FileResource fileResource = getFileResourceMetadata(trackedEntity, attribute, program);
     return FileResourceStream.ofImage(fileResourceService, fileResource, dimension);
   }
 
   private FileResource getFileResourceMetadata(
       UID trackedEntityUid, UID attributeUid, @CheckForNull UID programUid)
-      throws NotFoundException {
-    TrackedEntity trackedEntity = trackedEntityStore.getByUid(trackedEntityUid.getValue());
-    if (trackedEntity == null || trackedEntity.isDeleted()) {
-      throw new NotFoundException(TrackedEntity.class, trackedEntityUid.getValue());
-    }
+      throws NotFoundException, ForbiddenException {
+    TrackedEntity trackedEntity =
+        getNewTrackedEntity(
+            trackedEntityUid, programUid, TrackedEntityParams.FALSE.withIncludeAttributes(true));
 
     TrackedEntityAttribute attribute = getAttribute(attributeUid, trackedEntity, programUid);
```

Comment:
is the attribute logic regarding program vs tet attributes still needed in `getAttribute` now that we use `getNewTrackedEntity` passing in the program and retrieving the attributes?

Reply:
Removed all the logic about attributes from here and fix the code in `TrackedEntityAggregate`